UNITTESTS/moduletests/storage/kvstore/TDBStore/moduletest.cpp

```diff
@@ -0,0 +1,126 @@
+/* Copyright (c) 2019 ARM Limited
```

Comment:
2020

Reply:
+ once fixing this one, licenses to the new files below as well